src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -125,98 +115,123 @@ public EventValueInfo(final Class<E> event, final Class<T> c, final Getter<T, E>
 	 * It is recommended to use {@link EventValues#getEventValueGetter(Class, Class, int)} or {@link EventValueExpression#EventValueExpression(Class)} instead of invoking this
 	 * method repeatedly.
 	 * 
-	 * @param e
-	 * @param c
-	 * @param time
+	 * @param e event
+	 * @param c type of getter
+	 * @param time the event-value's time
 	 * @return The event's value
 	 * @see #registerEventValue(Class, Class, Getter, int)
 	 */
 	@Nullable
-	public static <T, E extends Event> T getEventValue(final E e, final Class<T> c, final int time) {
-		@SuppressWarnings({"null", "unchecked"})
-		final Getter<? extends T, ? super E> g = EventValues.getEventValueGetter((Class<E>) e.getClass(), c, time);
-		if (g == null)
+	public static <T, E extends Event> T getEventValue(E e, Class<T> c, int time) {
+		@SuppressWarnings("unchecked")
+		Getter<? extends T, ? super E> getter = getEventValueGetter((Class<E>) e.getClass(), c, time);
+		if (getter == null)
 			return null;
-		return g.get(e);
+		return getter.get(e);
 	}
 	
 	/**
 	 * Returns a getter to get a value from an event.
 	 * <p>
 	 * Can print an error if the event value is blocked for the given event.
 	 * 
-	 * @param e
-	 * @param c
-	 * @param time
+	 * @param e event class runtime calling

```

Comment:
Might be grammatical error here, please check. If it is not, please elaborate a bit instead. I'm not sure what this means.